pkg/cluster/spec.go

```diff
@@ -616,3 +616,88 @@ func (vb *VersionsBundle) Images() []v1alpha1.Image {
 
 	return images
 }
+
+func (vb *VersionsBundle) Ovas() []v1alpha1.Archive {
+	return []v1alpha1.Archive{
+		vb.EksD.Ova.Bottlerocket.Archive,
+		vb.EksD.Ova.Ubuntu.Archive,
+	}
+}
+
+func (vb *VersionsBundle) Manifests() map[string][]v1alpha1.Manifest {
+	manifests := map[string][]v1alpha1.Manifest{}
+
+	// CAPA manifests
+	manifests["cluster-api-provider-aws"] = append(
+		manifests["cluster-api-provider-aws"],
+		vb.Aws.Components,
+		vb.Aws.ClusterTemplate,
+		vb.Aws.Metadata,
+	)
```

Comment:
since we're initializing manifests at the start of this function can this just be:
```suggestion
	manifests["cluster-api-provider-aws"] = []v1alpha1.Manifest{
		vb.Aws.Components,
		vb.Aws.ClusterTemplate,
		vb.Aws.Metadata,
	}
```
same suggestion for rest of the components below

Reply:
Yeah sure. I was torn between doing this and the append approach but I'm cool with this too.